metrics/report/report_dockerfile/lifecycle-time.R

```diff
@@ -98,7 +98,8 @@ if (length(resultdirs) == 2) {
 	# This is a touch hard wired - but we *know* we only have two
 	# datasets...
 	for( i in 1:5) {
-		val = ((as.double(rstats[1,i]) / as.double(rstats[2,i])) * 100) - 100
+		diference = as.double(rstats[2,i]) - as.double(rstats[1,i])
```

Comment:
yay, this is looking good to me, and generated good output in my test. Just one tiny spelling niggle....
s/diference/difference/ throughout :-)